cli/cmd/root.go

```diff
@@ -86,6 +88,7 @@ func addProxyConfigFlags(cmd *cobra.Command, options *proxyConfigOptions) {
 	cmd.PersistentFlags().StringVar(&options.imagePullPolicy, "image-pull-policy", options.imagePullPolicy, "Docker image pull policy")
 	cmd.PersistentFlags().Int64Var(&options.proxyUID, "proxy-uid", options.proxyUID, "Run the proxy under this user ID")
 	cmd.PersistentFlags().StringVar(&options.proxyLogLevel, "proxy-log-level", options.proxyLogLevel, "Log level for the proxy")
+	cmd.PersistentFlags().StringVar(&options.proxyBindTimeout, "proxy-bind-timeout", "10s", "Timeout the proxy will use")
```

Comment:
This is super minor, but I think it would be better to use the initialized value of proxyBindTimeout here for the default, rather than setting a new default. So you could write this as:

```
	cmd.PersistentFlags().StringVar(&options.proxyBindTimeout, "proxy-bind-timeout", options.proxyBindTimeout, "Timeout the proxy will use")
```

That's more similar to how the rest of the flags in this func are handled.

Reply:
Yes you're totally right haven't even though about it !